scribble/app/views/home/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>RECENT POSTS</h1>
+<div class = "buttons">
+<a href="/posts">View All</a>
+  <%= link_to "Add New", new_post_path %>
+</div>
+<% @posts[0..2].each do |post| %>
```

Comment:
You can do `@recent_posts = Post.all.limit(3)` in the controller, and then use `@recent_posts` in the view here.

You might even sort by created_at first: `@recent_posts = Post.all.order(:created_at).limit(3)`